conda/base/constants.py

```diff
@@ -251,12 +251,16 @@ class SatSolverChoice(ValueEnum):
     PYSAT = 'pysat'
 
 
-class ExperimentalSolverChoice(ValueEnum):
+class SolverChoice(ValueEnum):
     CLASSIC = 'classic'
     LIBMAMBA = 'libmamba'
     LIBMAMBA_DRAFT = 'libmamba-draft'
 
 
+# TODO: Remove in a later release - compatibility alias
+ExperimentalSolverChoice = SolverChoice
```

Comment:
Do we need to mark this as PendingDeprecation too? How can we do it in a transparent way? Subclassing and overriding `__getattribute__` or similar?

Reply:
If this can be easily done, then let's do it.